tests/covalent_tests/workflow/transport_test.py

```diff
@@ -33,7 +33,7 @@
 def subtask(x):
     """Workflow subtask."""
 
-    return x ** 2
+    return x**2
```

Comment:
Binary operators must have a space separating them from both operands.

Reply:
pre-commit hooks did some weird stuff to some of the files... i'll fix this